app/src/main/java/yacs5e/ptpthingers/com/yacs5e_app/Background.java

```diff
@@ -0,0 +1,45 @@
+package yacs5e.ptpthingers.com.yacs5e_app;
+
+import java.util.List;
+
+/**
+ * Created by Kacper on 15.11.2017.
+ */
+
+public class Background {
+    private Feature Feature;
+    private List<String> PersonalityTrait;
+    private List<String> Proficiency;
```

Comment:
Could've named it "Proficiencies" instead since it's always more than one.
Good for now, tho.

Additional thought: Backgrounds sometimes allow you to choose proficiencies. Some proficiencies are in regard to skills, some to languages, some to tools/instruments/others. Maybe just a string list isn't the best solution.